instrumentation/lettuce/lettuce-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v4_0/LettuceDbAttributesExtractor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.lettuce.v4_0;
+
+import com.lambdaworks.redis.protocol.RedisCommand;
+import io.opentelemetry.instrumentation.api.instrumenter.db.DbAttributesExtractor;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+final class LettuceDbAttributesExtractor extends DbAttributesExtractor<RedisCommand<?, ?, ?>> {
+  @Override
+  protected String system(RedisCommand<?, ?, ?> request) {
+    return SemanticAttributes.DbSystemValues.REDIS;
```

Comment:
Or rather it looks like this class isn't used. Seems that we are seeing a pattern of annoyance in DB instrumentation since URI and statement are almost always different classes, unlike non-DB

Reply:
Nevermind, I didn't notice we have two INSTRUMENTER but see it now